fix: only allow EIP155 transaction (#244)
nervosnetwork/godwoken-web3: docs/compatibility.md

- Use the `transfer function` in [CKB_ERC20_Proxy](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b/solidity/erc20/README.md) contract [combined](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b322/solidity/erc20/SudtERC20Proxy_UserDefinedDecimals.sol#L154) with sUDT_ID = 1 (CKB a.k.a. pETH).

### Signing Transaction Only Support EIP155

> [EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. currently we only support EIP155 signing scheme.

Reviewer: ```suggestion
[EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. Currently we only support EIP155 signing scheme.
```

> ### Result

Reviewer: ```suggestion
### Known issue #238
```

Author: issue 234 can not be click if change to this?

> use outdated ethereum toolchain like `truffle-hdwallet-provider` to send transaction will result failed. 

Reviewer: ```suggestion
Using outdated Ethereum toolchain like `truffle-hdwallet-provider` to send transaction will result in failure. 
```

> issue: #238

Reviewer: ```suggestion
```
